src/frontend/webpack.config.js

```diff
@@ -92,7 +92,7 @@ module.exports = (env) => {
         },
         {
           exclude: /node_modules(?!\/richie-education)/,
-          test: new RegExp(`\.(tsx?|jsx?)$`),
+          test: /`\.(tsx?|jsx?)$`/,
```

Comment:
You have left backticks that is why build is broken
```suggestion
          test: /\.(tsx?|jsx?)$/,
```